packages/app-root/src/app/about/page.js

```diff
@@ -1,12 +1,15 @@
+'use client'
+import { usePathname } from 'next/navigation'
```

Comment:
Instead of using `asPath`, I switched to the new [next/navigation](https://nextjs.org/docs/app/api-reference/functions/use-router#migrating-from-nextrouter) hooks.

In order to render AboutHeader on this page, I had to remove any router-related hooks from AboutHeader and place them at the page level after encountering this error: [next-router-not-mounted](https://nextjs.org/docs/messages/next-router-not-mounted). Links from next/link can still be used in react components in lib-content though 🤷‍♀️